clarity/src/vm/representations.rs

```diff
@@ -455,12 +455,16 @@ pub struct SymbolicExpression {
     pub id: u64,
 
     #[cfg(feature = "developer-mode")]
+    #[serde(default)]
     pub span: Span,
 
     #[cfg(feature = "developer-mode")]
+    #[serde(default)]
     pub pre_comments: Vec<(String, Span)>,
     #[cfg(feature = "developer-mode")]
+    #[serde(default)]
     pub end_line_comment: Option<String>,
+    #[serde(default)]
     #[cfg(feature = "developer-mode")]
     pub post_comments: Vec<(String, Span)>,
```

Comment:
These could instead use `#[serde(default, skip_serializing_if = "Option::is_none")]` or `#[serde(default, skip_serializing_if = "Option::is_empty")]`. `Span` would require adding a new method... maybe an `is_zero`?

Reply:
I can give it a try, but i don't what would be the benefit?

Reply:
Just a preference to leaving those fields out versus serializing a default value.

Reply:
Maybe just leaving `Span` with default, since it is not optional, but the others could be omitted.

Reply:
In this case i'm setting a default value when deserializing.
The point is that the node api returns, as json, something like
```{"expre": "...", "id": 2}```

And when Clarinet needs to deserialize it, because it using clarity/developer-mode, it's able to parse it with, and set the default values (even though they are meaningless)